app/migrations/Version20180102140039.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types = 1);
```

Comment:
something went wrong here

Reply:
Should it go away? Not sure it makes a lot of difference on a migration though, but it's good to be strict on it.

Reply:
I just prefer to be coherent here. If you have a free moment I would love to merge a PR :)